### DISCUSSION TITLE
Cannot find module '@deepgram/sdk'

### DISCUSSION THREAD

**Discussion by hanyelmehy110** [deepgram/community#568] [General help]
### Which Deepgram product are you using?

None

### Details

when try to install on my vps i get error Cannot find module '@deepgram/sdk'

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@hanyelmehy110  :wav Can you give us more details here so we can try to assist? 

* What SDK and version are you trying to use?
* Can you provide some sample code so we can better understand what you are trying to do.
* When you say  "when try to install on my vps" Can you elaborate on what this is?
